src/wizards.ts

```diff
@@ -0,0 +1,250 @@
+import { ListItem } from '@material/mwc-list/mwc-list-item';
+import { html } from 'lit-element';
+import { repeat } from 'lit-html/directives/repeat';
+import { get, translate } from 'lit-translate';
+
+import {
+  EditorAction,
+  identity,
+  isEqual,
+  isSame,
+  newWizardEvent,
+  SimpleAction,
+  Wizard,
+  WizardAction,
+} from './foundation.js';
+
+interface MergeOptions {
+  title?: string;
+  selected?: (diff: Diff<Element | string>) => boolean;
+}
+
+export type Diff<T> =
+  | { ours: T; theirs: null }
+  | { ours: null; theirs: T }
+  | { ours: T; theirs: T };
+
+function describe(element: Element): string {
+  const id = identity(element);
+
+  return typeof id === 'string' ? id.replace(/^>/, '') : get('unidentifiable');
+}
+
+function mergeWizardAction(
+  attrDiffs: [string, Diff<string>][],
+  childDiffs: Diff<Element>[],
+  sink: Element,
+  source: Element,
+  options?: MergeOptions
+): WizardAction {
+  return (_, wizard: Element): EditorAction[] => {
+    const actions: SimpleAction[] = [];
+    const checkList = wizard.shadowRoot!.querySelector('mwc-list')!;
+
+    const selectedAttrDiffs = (<ListItem[]>checkList.selected)
+      .filter(item => item.classList.contains('attr'))
+      .map(item => attrDiffs[(item.value as unknown) as number]);
+
+    if (selectedAttrDiffs.length) {
+      const newSink = <Element>sink.cloneNode(false);
+      if (sink.childElementCount === 0) newSink.textContent = sink.textContent;
+      for (const [name, diff] of selectedAttrDiffs)
+        if (name === 'value') {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7195001)
```suggestion
        if (name === "value") {
```